bin/topological_inventory-persister

```diff
@@ -13,9 +13,7 @@ TopologicalInventory::Core::ArHelper.load_environment!
 persister_worker = TopologicalInventory::Persister::Worker.new(:host => queue_host, :port => queue_port)
 
 begin
-  Thread.new { persister_worker.run }
-
-  loop { sleep 5 }
+  persister_worker.run
 rescue Interrupt
```

Comment:
Doesn't that make this rescue a bit redundant given the `ensure` block in the persister?

Reply:
:+1: yeah it does

Reply:
Don't need `.stop` either now